lib/matrix/matrix.rb

```diff
@@ -19,12 +19,39 @@ class Matrix
     alias element []
     alias component []
 
+
+    def get_rows
```

Comment:
`get_<something>` isn't Ruby style, Please, rename to `rows` and make `private`.